backend/templates/base.html

```diff
@@ -0,0 +1,41 @@
+{% load cms_tags menu_tags sekizai_tags %}
+<DOCTYPE html>
+<html lang = "{{ LANGUAGE_CODE}}">
+
+  <head>
+    <meta name="description" content="{% page_attribute 'meta_description' %}">
+    <title>{% page_attribute "page_title" %}</title>
+    {% render_block "css" %}
+  </head>
+
+  <body>
+    {% cms_toolbar %}
+    <header class="header">
+
+      <div class="lang-select">
+          {% language_chooser %}
+      </div>
+
+      <div class="logo">
+        <a href="/sv"><img
+        src="/assets/img/zf-logo-red.svg" alt="ZetkinLogo"></a>
+      </div>
+
+      <div class="menu">
+        <ul class="nav navbar-nav">
+          {% show_menu 0 100 100 100 %}
+        </ul>
+      </div>
+    </header>
+
+    <!-- Child template content block is automatically placed between these two -->
```

Comment:
HTML comments are included in the rendered HTML so it's not a good idea to use them to document template logic. This line should be removed altogether.